.travis.yml

```diff
@@ -3,12 +3,14 @@ go_import_path: "github.com/juju/simplekv"
 go: 
   - "1.9"
   - "1.10"
+  - "1.11.5"
```

Comment:
how about just removing 1.9 and 1.10 support, make this just "1.11" and remove the dependencies.tsv file?
AFAICS all the places it's being used are already using modules.